tools/webpack.config.babel.js

```diff
@@ -4,14 +4,27 @@ const AssetsPlugin = require('assets-webpack-plugin')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
 const { CleanWebpackPlugin } = require('clean-webpack-plugin')
 const CopyPlugin = require('copy-webpack-plugin')
+const MomentTimezoneDataPlugin = require('moment-timezone-data-webpack-plugin')
+const MomentLocalesPlugin = require('moment-locales-webpack-plugin')
 const babelConfig = require('../.babelrc.js')
 const fs = require('fs')
 
 const SHORT_COMMIT_SHA_LENGTH = 8
 
+// A first performance budget, which should be improved (= reduced) in the future: Maximum bundle size in MiB
+// eslint-disable-next-line no-magic-numbers
+const MAX_BUNDLE_SIZE = 1.5 * Math.pow(2, 20)
+
+const readJson = path => JSON.parse(fs.readFileSync(path))
+
 const readVersionName = () => {
-  const versionFile = fs.readFileSync(path.resolve(__dirname, '../version.json'))
-  return JSON.parse(versionFile).versionName
+  const versionFile = readJson(path.resolve(__dirname, '../version.json'))
+  return versionFile.versionName
+}
+
+const getSupportedLocales = () => {
+  const localesConfig = readJson(path.resolve(__dirname, '../locales/config.json'))
+  return [localesConfig.sourceLanguage, ...localesConfig.targetLanguages]
```

Comment:
Did you check whether the config is uptodate and contains all languages?

Reply:
It contains all languages which should be translated I guess. Alternatively we could also determine this information by just looking for the root-keys of the `locales.json`. What do you think?

Reply:
I don't think there should be a difference between the keys in the localesConfig and the root keys of the locales.json as I don't see a reason why we would want to have languages in the locales.json that should not be translated?

Reply:
Using that config is fine. This config is used by the manage script. There can not be more or less languages in our locales.json

Reply:
So it's auto generated? I think the manage script is also executed if new translations are to be made, so if we would manually add a language, it would be missing here until the next translation process. However, this should be fine as that probably won't happen.

Reply:
Yeah that will probably not happen. All our translations for new languages come from external services